docs: add rdws secrets (#4494)
aws/copilot-cli: site/content/blogs/release-v126.en.md

## Sidecar image build

## Request-Driven Web Service secrets support

> You can now add your secrets(SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:

Reviewer: ```suggestion
You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:
```

> You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. First ensure that your secrets are tagged correctly. Then simply update your Request-Driven Web Service manifest with:

Reviewer 2: ```suggestion
You can now add your secrets (from SSM Parameter Store or AWS Secrets Manager) to your App Runner service as environment variables using Copilot. 

Similar to other service types such as Load-Balanced Web Service, you need to first add the following tags to your secrets:
| Key                     | Value                                                       |
| ----------------------- | ----------------------------------------------------------- |
| `copilot-application`   | Application name from which you want to access the secret   |
| `copilot-environment`   | Environment name from which you want to access the secret   |

Then simply update your Request-Driven Web Service manifest with:
```

Author: Much Better! Thanks.

```yaml
  secrets:

>     secret-name-as-env-variable: secret-name-from-aws

Reviewer 2: ```suggestion
    GITHUB_TOKEN: GH_TOKEN_SECRET
```

```
And deploy! Your service can now access the secret as an environment variable.

> For more detailed use of the secrets field:

Reviewer: ```suggestion
For more detailed use of the `secrets` field:
```

> And deploy! Your service can now access the secret as an environment variable.
> For more detailed use of the `secrets` field:

Reviewer 2: One "\n" will not render as a new line in md. We need two consecutive \n:
```suggestion
And deploy! Your service can now access the secret as an environment variable.

For more detailed use of the `secrets` field:
```

```yaml
secrets:

>   # SecretsManager

Reviewer 2: ```suggestion
  # To inject a secret from SecretsManager.
```

  # (Recommended) Option 1. Referring to the secret by name.
  DB:
    secretsmanager: 'demo/test/mysql'
  # You can refer to a specific key in the JSON blob.
  DB_PASSWORD:
    secretsmanager: 'demo/test/mysql:password::'
  # You can substitute predefined environment variables to keep your manifest succinct.
  DB_PASSWORD:
        secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/mysql:password::'

  # Option 2. Alternatively, you can refer to the secret by ARN.
  DB: 'arn:aws:secretsmanager:us-west-2:111122223333:secret:demo/test/mysql-Yi6mvL'
 

>   #SSM Parameter Store

Reviewer 2: ```suggestion
  # To inject a secret from SSM Parameter Store
```

>   # (Recommended) Option 1. Referring to the secret by ARN.

Reviewer 2: Why is this recommended? I would probably have recommended option 2 😂

Author: :P At App Runner side we mostly use ARNs. Should I remove the `Recommended` word ?

Reviewer 2: Yes let's remove "Recommended" 👍🏼 !

Author: Done!

  GITHUB_WEBHOOK_SECRET: 'arn:aws:ssm:us-east-1:615525334900:parameter/GH_WEBHOOK_SECRET'

  # Option 2. Referring to the secret by name.
  GITHUB_WEBHOOK_SECRET: GITHUB_WEBHOOK_SECRET
```

> For more details see (../../docs/manifest/rd-web-service/#secrets-from-cfn).

Reviewer 2: ```suggestion
See the [manifest specification](../../docs/manifest/rd-web-service/#secrets). To learn more about injecting secrets into your services, see [the secrets page](../../docs/developing/secrets.en.md)
```

Author: Great!

Similar to other service types such as Load-Balanced Web Service, you need to first add the following tags to your secrets:
| Key                     | Value                                                       |
| ----------------------- | ----------------------------------------------------------- |
| `copilot-application`   | Application name from which you want to access the secret   |
| `copilot-environment`   | Environment name from which you want to access the secret   |

Then simply update your Request-Driven Web Service manifest with:
```yaml
  secrets:
    GITHUB_TOKEN: GH_TOKEN_SECRET
```
And deploy! Your service can now access the secret as an environment variable.

For more detailed use of the `secrets` field:
```yaml
secrets:
  # To inject a secret from SecretsManager.
  # (Recommended) Option 1. Referring to the secret by name.
  DB:
    secretsmanager: 'demo/test/mysql'
  # You can refer to a specific key in the JSON blob.
  DB_PASSWORD:
    secretsmanager: 'demo/test/mysql:password::'
  # You can substitute predefined environment variables to keep your manifest succinct.
  DB_PASSWORD:

>         secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/mysql:password::'

Author: Indentation
